MetaMask/metamask-extension
# Add support for Polygon

## Explanation
Support for Polygon in Swaps

## Manual testing steps
- Add Polygon Mainnet in the extension (https://docs.matic.network/docs/develop/network-details/network/)
- Click on `Swap` on the main page (make sure that you have enough funds to do swapping)
- Select `Swap from` and `Swap to` values and do swapping
- See the success page
